composer.json

```diff
@@ -10,7 +10,8 @@
     "cakephp/plugin-installer": "*"
   },
   "require-dev": {
-    "phpunit/phpunit": "4.1.*"
+    "phpunit/phpunit": "4.1.*",
+    "loadsys/loadsys_codesniffer": "~3.0"
```

Comment:
This isn't required. The **composer.json** is only used during `create-project`. The **composer.json.template** will be transformed into the composer.json "proper" for the generated app, so the `.template` version is the only thing we need to keep up to date. I'm reminded of a comment I made... (https://github.com/loadsys/CakePHP-Skeleton/issues/70#issuecomment-72271709)